app/views/needs/_form.html.erb

```diff
@@ -1,92 +1,168 @@
-    %>
-    <%= f.input :other_evidence,
-      :label => "Do you have any other evidence to support this need? <br/><span class='optional'>(optional)</span>".html_safe,
-      :hint => "E.g. user research, campaigns, user demand.",
-      :as => :text,
-      :input_html => { :class => 'input-md-8', :rows => 3 }
-    %>
-  <% end %>
 
-  <%= f.input :impact,
-    :label => "What is the impact of GOV.UK not doing this? <br/><span class='optional'>(optional)</span>".html_safe, :hint => "What is the impact of GOV.UK not doing this?",
-    :as => :radio,
-    :collection => Need::IMPACT
-  %>
+    <div class="form-group">
+      <span class="form-label">
+        <%= f.label :legislation, "What legislation underpins this need? <br/><span class='optional'>(optional)</span>".html_safe %>
+      </span>
+      <span class="form-wrapper">
+        <%= f.text_area :legislation, rows: 3,  class: "input-md-8 form-control" %>
+        <span class="help-block">Please supply a URL, e.g. on legislation.gov.uk</span>
+      </span>
+    </div>
+
+    <div class="form-group">
+      <span class="form-label">
+        <%= f.label :yearly_user_contacts, "Roughly how many user contacts do you get about this need per year <br/><span class='optional'>(optional)</span>".html_safe %>
+      </span>
+      <span class="form-wrapper">
+        <%= f.text_field :yearly_user_contacts, class: "input-md-2 form-control" %>
+        <span class="help-block">Include calls to contact centres, emails, customer service tickets, etc.</span>
+      </span>
+    </div>
+
+    <div class="form-group">
+      <span class="form-label">
+        <%= f.label :yearly_need_views, "Pageviews specific to this need per year <br/><span class='optional'>(optional)</span>".html_safe %>
+      </span>
+      <span class="form-wrapper">
+        <%= f.text_field :yearly_need_views, class: "input-md-2 form-control" %>
+        <span class="help-block">Combine the pageviews for all relevant content.</span>
+      </span>
+    </div>
+
+    <div class="form-group">
+      <span class="form-label">
+        <%= f.label :yearly_site_views, "Pageviews for your website per year <br/><span class='optional'>(optional)</span>".html_safe %>
+      </span>
+      <span class="form-wrapper">
+        <%= f.text_field :yearly_site_views, class: "input-md-2 form-control" %>
+        <span class="help-block">This is for your whole website, not just this user need.</span>
+      </span>
+    </div>
+
+    <div class="form-group">
+      <span class="form-label">
+        <%= f.label :yearly_searches, "How many searches relevant to this need are carried out per year <br/><span class='optional'>(optional)</span>".html_safe %>
+      </span>
+      <span class="form-wrapper">
+        <%= f.text_field :yearly_searches, class: "input-md-2 form-control" %>
+        <span class="help-block">Include external and internal searches.</span>
+      </span>
+    </div>
```

Comment:
Just wondered if something needed to be done with `:pattern =>"[0-9]*"` from the validation in the formtastic form. Seems like a regex only allowing numbers?

Reply:
Ah yes, thanks :D